shared_model/bindings/model_query_builder_embind.cpp

```diff
@@ -0,0 +1,66 @@
+#include <emscripten/bind.h>
+#include <emscripten/val.h>
+
+#include "model_proto.hpp"
+#include "model_query_builder.hpp"
+
+using namespace emscripten;
+using namespace shared_model;
+using namespace shared_model::bindings;
+using namespace shared_model::proto;
+
+using namespace shared_model::crypto;
+
+EMSCRIPTEN_BINDINGS(model_query_builder)
+{
+  register_vector<Hash>("HashVector");
+  
+  class_<Hash, base<Blob>>("Hash")
+  .constructor<const std::string&>()
+  .function("toString", &Hash::toString)
+  ;
+
+  class_<Query>("Query")
+  .function("creatorAccountId", &Query::creatorAccountId)
+  .function("queryCounter", &Query::getQueryCounter)
+  .function("blob", &Query::blob)
+  .function("payload", &Query::payload)
+  .function("addSignature", &Query::addSignature)
+  .function("createdTime", &Query::getCreatedTime)
+  ;
```

Comment:
signatures? :(
I guess it's not that trivial so add a todo plz (also for the other proto model types)